src/assets/scripts/client/scope/RadarTargetCollection.js

```diff
@@ -0,0 +1,214 @@
+import _filter from 'lodash/filter';
+import _forEach from 'lodash/forEach';
+import _has from 'lodash/has';
+import _isObject from 'lodash/isObject';
+import RadarTargetModel from './RadarTargetModel';
+import BaseCollection from '../base/BaseCollection';
+import EventBus from '../lib/EventBus';
+import { EVENT } from '../constants/eventNames';
+import { THEME } from '../constants/themes';
+
+/**
+ * Collection of `RadarTargetModel`s
+ *
+ * @class RadarTargetCollection
+ */
+export default class RadarTargetCollection extends BaseCollection {
+    constructor(theme) {
+        super();
+
+        this._eventBus = EventBus;
+        this._theme = theme;
+
+        this._init()
+            .enable();
+    }
+
+    /**
+     * Return a read-only reference to the collection array
+     *
+     * @for RadarTargetCollection
+     * @property items
+     * @type {array}
+     */
+    get items() {
```

Comment:
isn't this already defined in the `BaseCollection`?

Reply:
Negative, it is not. :confused: